test/correctness/document-correctness.py

```diff
@@ -365,9 +364,8 @@ def _run_operation_(op1, op2):
 
         if not indexes_first:
             for i in indexes:
-                # When we do enable make sure we don't enable for 50% cases. That way unique indexes will cloud everything else. Make it something like 5-10%.
-                # uniqueIndex = gen.global_prng.choice([True, False])
-                uniqueIndex = False
+                # 5% likelyhood to be unique, assuming a uniform distribution
+                uniqueIndex = (gen.global_prng.randint(1,20) == 1)
```

Comment:
We are trying to recreate the same indexes that might have created above. As we discussed Doc Layer ignores index recreation with the same name. But, with this change, its possible first time it was created as a normal index and later here as a unique index or vice versa. Its probably is fine but could generate weird cases in the future. Can unique index choice be made part of `gen.random_index_spec()`? If we do that, now that function could return either a tuple like `(indexkeys_array, options_map)` or an IndexSpec object.